src/components/ADempiere/Form/VPOS/Collection/index.vue

```diff
@@ -225,7 +251,12 @@
                         :currency="pointOfSalesCurrency"
                       />
                       <el-button slot="reference" type="text" style="color: #000000;font-weight: 604!important;font-size: 100%;">
-                        {{ formatPrice(pay, pointOfSalesCurrency.iSOCode) }}
+                        {{
```

Comment:
Just can see change for source code with the same functionality, please let it without changes